src/components/ContentNavigator/Sidebar.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import {
+  Box,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemText,
+  Typography,
+} from "@mui/material";
+
+interface SidebarProps {
+  sections: { id: string; title: string }[];
+  onScrollToSection: (id: string) => void;
+}
+
+const Sidebar: React.FC<SidebarProps> = ({ sections, onScrollToSection }) => {
+  return (
+    <Box width="20%" bgcolor="#1a1a1a" color="white" p={2}>
```

Comment:
For the bgcolor, use the colors in styleCOnstant, like:
`<Typography color={COLORS.GREEN} >HOLAAAA</Typography>`

Reply:
Sure